src/Command/ServeCommand.php

```diff
@@ -70,29 +69,25 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             (new Dotenv())->load($rootDir.\DIRECTORY_SEPARATOR.$envFile, ...$extra);
```

Comment:
We need to check if dotenv exists. 

Now we load dependencies both here and in the helper